app/controllers/admin/users_controller.rb

```diff
@@ -10,6 +10,19 @@ def index
       @users = User.all
     end
 
+    def toggle_confirmation
+      if params[:user][:should_be_confirmed] == 'true'
+        params[:user].delete(:should_be_confirmed)
+        params[:user][:confirmed_at] = Time.now
+        @user.update_attributes(user_params)
+      elsif params[:user][:should_be_confirmed] == 'false'
+        params[:user].delete(:should_be_confirmed)
```

Comment:
No you cannot. Did you read about strong parameters? The logic might not apply 100% but I would like that we still whitelist the parameter we want to be using and call user_params instead of calling parameterers directly on Lines 14 and 18.